q2cli/core/cache.py

```diff
@@ -232,7 +232,7 @@ def _get_current_state(self):
         import qiime2.sdk
 
         state = {
-            'plugins': {}
+            'plugins': {},
```

Comment:
Please revert this, since we didn't wind up making any meaningful changes to the cache.